src/index.ts

```diff
@@ -1,14 +1,17 @@
+import {AEGTree} from "./AEG/AEGTree";
+import {CutNode} from "./AEG/CutNode";
+import {Ellipse} from "./AEG/Ellipse";
+import {AtomNode} from "./AEG/AtomNode";
+import {ellipseCreation, removeCutListener} from "./EllipseCreation";
+import {atomCreation, removeAtomListener} from "./AtomCreation";
+
 /**
  * A program to draw ellipses and atoms.
  * @author Dawn Moore
  * @author James Oswald
  */
 
```

Comment:
Keep file comment above imports.  
```suggestion
/**
 * A program to draw ellipses and atoms.
 * @author Dawn Moore
 * @author James Oswald
 */

import {AEGTree} from "./AEG/AEGTree";
import {CutNode} from "./AEG/CutNode";
import {Ellipse} from "./AEG/Ellipse";
import {AtomNode} from "./AEG/AtomNode";
import {ellipseCreation, removeCutListener} from "./EllipseCreation";
import {atomCreation, removeAtomListener} from "./AtomCreation";
```